ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParseProxyTest.java

```diff
@@ -40,6 +40,10 @@ public void testOutput() {
             + "{\"submission_timestamp\":\"2000-01-01T00:00:00.000000Z\"" //
             + ",\"x_forwarded_for\":\"3, 2, 1\"" //
             + "},\"payload\":\"notProxied++\"}",
+        "{\"attributeMap\":" //
+            + "{\"submission_timestamp\":\"2000-01-01T00:00:00.000000Z\"" //
+            + ",\"x_forwarded_for\":\"4, 3, 6.7.8.9, 1\"" //
+            + "},\"payload\":\"statroxied++\"}",
```

Comment:
@relud I was getting errors when I tried to put a better name in the payload. I believe this is getting interpreted as a base64-encoded value, so it's kind of a crapshoot as to whether the nice name you put here is going to work or not. Is that true? Any advice on what to put in the payload here?

Reply:
payload is definitely base64 encoded, the `+`s are being used for padding to make it valid base64, so in this case I'd use:
```suggestion
            + "},\"payload\":\"staticProxied+++\"}",
```